micro-ui/web/micro-ui-internals/packages/libraries/src/translations/index.js

```
@@ -31,11 +31,11 @@ const i18nextConfig = () => ({
 });
 
 function replaceLiterals(text = "", dynamicValues = {}) {
-  let returnText = text;
+  let returnText = String(text);
   const regex = /[^\{\{][\{]\w+/;
   if (regex.exec(text) !== null) {
     Object.keys(dynamicValues).forEach((key) => {
-      returnText = returnText.replace(`{${key.toUpperCase()}}`, dynamicValues[key]);
+      returnText = returnText && returnText?.replace(`{${key.toUpperCase()}}`, dynamicValues[key]);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)